### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

Note: the examples for `pandas.HDFStore.groups` and `pandas.HDFStore.walk` were done [here](https://github.com/pandas-dev/pandas/pull/53889).

### CODE REVIEW

File: pandas/core/resample.py [hunk 1]
```diff
@@ -503,6 +503,53 @@ def ffill(self, limit: int | None = None):
...
+        dtype: int64
+
+        Example for ``ffill`` with downsampling (we have fewer dates after resampling):
+
+        >>> ser.resample('M').ffill()
+        2023-01-31    2
+        2023-02-28    4
+        Freq: M, dtype: int64
+
+        Example for ``ffill`` with upsampling (we have more dates after resampling):
```
Comment (mroeschke) [inline pandas/core/resample.py:527]:
```suggestion
        Example for ``ffill`` with upsampling (fill the new dates with the previous value):
```

Might be a little more clear

File: pandas/core/resample.py [hunk 2]
```diff
@@ -503,6 +503,53 @@ def ffill(self, limit: int | None = None):
...
+        2023-01-08    1
+        2023-01-15    2
+        2023-01-22    2
+        2023-01-29    2
+        2023-02-05    3
+        2023-02-12    3
+        2023-02-19    4
+        Freq: W-SUN, dtype: int64
+
+        With upsampling and limiting (we have more dates after resampling but
```
Comment (mroeschke) [inline pandas/core/resample.py:540]: Then here the commentary can be `(only fill the first new date with the previous value)`

File: pandas/core/resample.py (hunk 1 above)
Comment (DeaMariaLeon) [reply pandas/core/resample.py]: Yes, thank you.

File: pandas/core/resample.py (hunk 2 above)
Comment (DeaMariaLeon) [reply pandas/core/resample.py]: Done.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @DeaMariaLeon

**Pull request merged**